system/lib/compiler-rt/stack_ops.S

```diff
@@ -3,37 +3,45 @@
 .globl stackAlloc
```

Comment:
I would have imagined that renaming this file to the same name with a different case might confuse windows .. but if the CI passes then I guess we are all good.

Reply:
It did confuse Windows git, who didn't pick up the rename initially :)